pages/common/iverilog.md

```diff
@@ -0,0 +1,24 @@
+# iverilog
+
+> Preprocesses and compiles Verilog HDL (IEEE-1364) code, into executable programs for simulation.
+> More information: <http://iverilog.icarus.com/>.
+
+- Compile a source file into an executable:
+
+`iverilog {{source.v}} -o {{executable}}`
+
+- Also display all warnings:
+
+`iverilog {{source.v}} -Wall -o {{executable}}`
+
+- Compile and run explicitly using the VVP runtime:
+
+`iverilog -o {{execuable}} -tvvp {{source.v}}`
+
+- Compile using Verilog library files from a different path:
+
+`iverilog {{source.v}} -o {{executable}} -I{{path/to/library_directory}}`
```

Comment:
Isn't that "-I{{path/to/library_directory}}" missing a space?

Reply:
Almost any program which takes single letter options only optionally requires a space between the option letter and the value. For example:

![image](https://user-images.githubusercontent.com/14198070/66141866-befcb180-e604-11e9-9e38-5a2f5ad93ee6.png)

The variant without space is very commonly used in compilers for `-I` (include path) and `-L` (library path). See for example our pages for [`gcc`](https://github.com/tldr-pages/tldr/tree/master/pages/common/gcc.md), [`g++`](https://github.com/tldr-pages/tldr/tree/master/pages/common/g++.md), [`clang`](https://github.com/tldr-pages/tldr/blob/master/pages/common/clang.md], and so on:

    - Include libraries from a different path:

    `gcc {{source.c}} -o {{executable}} -I{{header_path}} -L{{library_path}} -l{{library_name}}`

Reply:
Thank you @mebeim 

That being said, the PR looks good to me.